components/dashboard/src/settings/PersonalAccessTokens.tsx

```diff
@@ -534,7 +534,7 @@ function ListAccessTokensView() {
         <>
             <div className="flex items-center sm:justify-between mb-4">
                 <div>
-                    <h3>Personal Access Tokens</h3>
+                    <h3>Access Tokens</h3>
```

Comment:
**nitpick**: Since you're changing this, could we also update the empty state and drop the `PAT` acronym?

https://github.com/gitpod-io/gitpod/blob/d4f97802102e0c45e61e25a0bd8af6f42e305167/components/dashboard/src/settings/PersonalAccessTokens.tsx#L605

Reply:
Done.

Reply:
Thanks, @easyCZ! 🍊